unittest/cp_test_results.txt

```diff
@@ -6,20 +6,20 @@
 0.9932249481447798
 0.6040159648938821
 0.5862586400695924
-0.0562044207525656
-0.183409953450329
+0.605047103544661
```

Comment:
the original results were wrong?

Reply:
Yes the original "tucker compression" algorithm in CP-ALS class is not a valid approach. Possibly it should be deprecated?  These values have been modified to use the new `TUCKER_CP_ALS` class which is a mathematically correct approach.